src/helpers/url-helpers.ts

```diff
@@ -26,6 +23,11 @@ export function isValidExternalURL(url: string | URL) {
   return isAllowed;
 }
 
+export function fixUrl(url: string | URL) {
+  return url.toString().replaceAll('//', '/').replace('http:/', 'http://').replace('https:/', 'https://').replace('file:/', 'file://');
```

Comment:
I think this isn't needed now as we store the url correctly :-)

Reply:
yes - but, i felt its a safety net in case of new code where there's `//`. Do you still want me to remove it? I would prefer to keep it for now